src/topology/topological_fiber_bundle.lean

```diff
@@ -811,6 +821,10 @@ different name for typeclass inference. -/
 @[nolint unused_arguments, reducible]
 def total_space := bundle.total_space Z.fiber
 
+/-- Constructor for the total space of a `topological_fiber_bundle_core`. -/
```

Comment:
Why not introducing it for a general bundle, i.e., `total_space_mk E b a` (with the specific use `total_space_mk Z.fiber b a`)?